### PR TITLE
Use a consistent version for ScanCode and its imported license text resources

### PR DESCRIPTION
**Pull request by fviernau**
For clarity and consistency.

### CODE REVIEW

File: spdx-utils/build.gradle.kts [hunk 1]
```diff
@@ -72,7 +74,7 @@ val importSpdxLicenseTexts by tasks.registering(SvnExport::class) {
     description = "Imports license texts from the SPDX repository."
     group = "SPDX"
 
-    svnUrl = "https://github.com/spdx/license-list-data/trunk/text"
+    svnUrl = "https://github.com/spdx/license-list-data/z/text"
```
Comment (mnonnenmacher) [inline spdx-utils/build.gradle.kts:77]: What's the meaning of this change?

Comment (fviernau) [reply spdx-utils/build.gradle.kts]: This change was not intended. Removed it.

**Review by mnonnenmacher — COMMENTED** [review]
I'm surprised there was only a single license deleted in the last comment, I would have expected more changes since 3.2.1-rc2. Are you sure everything is working correctly?

**Comment by fviernau** [comment]
> I'm surprised there was only a single license deleted in the last commit, I would have expected more changes since 3.2.1-rc2. Are you sure everything is working correctly?

The 3.2.1-rc2 release has been created on 2020-09-11 [1] while the last time these text results were updated was on
2020-10-08 [2]. Since that time interval is quite small I wouldn't expect too many changes so I'm not surprised. I haven't double checked it against ScanCodes' Git history though.

[1] https://github.com/nexB/scancode-toolkit/releases/tag/v3.2.1rc2
[2] https://github.com/oss-review-toolkit/ort/commit/f6b7ab9d1bfd89b1c968db5f9896e90d834982d6

**Comment by mnonnenmacher** [comment]
> > I'm surprised there was only a single license deleted in the last commit, I would have expected more changes since 3.2.1-rc2. Are you sure everything is working correctly?
> 
> The 3.2.1-rc2 release has been created on 2020-09-11 [1] while the last time these text results were updated in ORT was on
> 2020-10-08 [2]. Since that time interval is quite small I wouldn't expect too many changes so I'm not surprised. I haven't double checked it against ScanCodes' Git history though.
> 
> [1] https://github.com/nexB/scancode-toolkit/releases/tag/v3.2.1rc2
> [2] [f6b7ab9](https://github.com/oss-review-toolkit/ort/commit/f6b7ab9d1bfd89b1c968db5f9896e90d834982d6)

Thanks for checking, I thought we would update more frequently.

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**